Core/Extensions/CredentialProviderExtensions.cs

```diff
@@ -1,48 +0,0 @@
-﻿using System;
-using System.Net;
-
-namespace NuGet
-{
-    internal static class CredentialProviderExtensions
```

Comment:
was this dead code?

Reply:
seemed to be, nothing broke once it was gone :)